force-app/main/default/classes/TriggerFactory.cls

```diff
@@ -0,0 +1,41 @@
+public class TriggerFactory {
+	public class TriggerException extends Exception {
+	}
+	public static void execute(SObjectType sobjectType) {
+		ITriggerHandler handler = createHandler(sobjectType);
+		handler.disableTrigger();
```

Comment:
why do we use here disableTrigger() and enableTrigger() methods?

Reply:
we use here disableTrigger() and enableTrigger() methods to check operability